<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sim_U4_T3</class>
 <widget class="QWidget" name="sim_U4_T3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sim_U4_T3</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#sim_U4_T3,
QScrollArea#scrollArea{
background-color:rgba(0,0,0,0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>284</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#scrollAreaWidgetContents{
background-color:rgb(0,0,0);
}
QLabel{
color:white;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Operadores LÃ³gicos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Imagen Original</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Imagen Resultante</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="CustomPlotWidget" name="cp_1" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="CustomPlotWidget" name="cp_2" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>(</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_imgBin_1">
           <item>
            <property name="text">
             <string>Rojo Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Verde Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Azul Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOT Rojo Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOT Verde Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOT Azul Binaria</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_opLog_1">
           <item>
            <property name="text">
             <string>AND</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>XOR</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_imgBin_2">
           <item>
            <property name="text">
             <string>Rojo Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Verde Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Azul Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOT Rojo Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOT Verde Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOT Azul Binaria</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_opLog_2">
           <item>
            <property name="text">
             <string>AND</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>XOR</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_imgBin_3">
           <item>
            <property name="text">
             <string>Ignorar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rojo Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Verde Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Azul Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOT Rojo Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOT Verde Binaria</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOT Azul Binaria</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="CustomPlotWidget" name="cp_5" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Azul &gt;10% Binaria</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Rojo &gt;50% Binaria</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="CustomPlotWidget" name="cp_4" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="CustomPlotWidget" name="cp_3" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Verde &gt;50% Binaria</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>NOT Rojo &gt;50% Binaria</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="CustomPlotWidget" name="cp_6" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>NOT Verde &gt;50% Binaria</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>NOT Azul &gt;10% Binaria</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="CustomPlotWidget" name="cp_7" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="CustomPlotWidget" name="cp_8" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPlotWidget</class>
   <extends>QWidget</extends>
   <header>clases.PlotWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
